### PR TITLE
Breadcrumb-track

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Wenn man in der Breadcrumb Navi der Layer Collection Liste navigiert, dann wird die Selected Collection nicht neu emittet](https://trello.com/c/uTnRsKGq/1786-wenn-man-in-der-breadcrumb-navi-der-layer-collection-liste-navigiert-dann-wird-die-selected-collection-nicht-neu-emittet)

File: projects/common/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts [hunk 1]
```diff
@@ -172,6 +172,7 @@ export class LayerCollectionNavigationComponent implements OnInit, OnChanges, On
 
     protected updateListView(id?: LayerCollectionItemOrSearch): void {
         this.selectedCollection = id ?? {type: 'collection', id: this.collectionId};
+        this.navigateCollection.emit(this.selectedCollection as LayerCollectionItem);
```
Comment (ChristianBeilschmidt) [inline projects/common/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts:175]: What does updateListView do in the first place?

Why does it emit something now? (despite the bug)

Comment (ChristianBeilschmidt) [reply projects/common/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts:175]: müsste das nicht in die navigate-Methode dann?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/common/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts (hunk 1 above)
Comment (michaelmattig) [reply projects/common/src/lib/layer-collections/layer-collection-navigation/layer-collection-navigation.component.ts]: I tried to separate it a bit better but I don't really want to touch any of the breadcrumb stuff I don't have to 5a41345c365156cb3a8ccbfb6bb41ebf9a4cab2d

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**